### PR TITLE
Renaming search to search-books, adding e2e tests and circleci configuration

### PR DESCRIPTION
**Pull request by mverdaguer**

### CODE REVIEW

**Review by MarcPuigdemont — APPROVED** [review]

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+          name: "Setup orient-express"
+          command: |
+            cd orient-express
+            npm install
+            echo -n 'GOODREADS_KEY=' >> .env
+            echo $GOODREADS_KEY >> .env
+            echo -n 'GOODREADS_SECRET=' >> .env
+            echo $GOODREADS_SECRET >> .env
+            echo -n 'OMBD_KEY=' >> .env
+            echo $OMBD_KEY >> .env
```
Comment (marcmortensen) [inline .circleci/config.yml:40]: Would love to have a more clever way to solve this, LGMT thought.

File: src/views/Books.vue [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <template>
-  <v-container grid-list-sm fluid>
+  <v-container id="books-container" grid-list-sm fluid>
```
Comment (marcmortensen) [inline src/views/Books.vue:2]: i get the usage of id here (for the e2e test), my only problem if we ever reuse it we could potentially have duplicated id's :s, we should create a ticket for studying which solution would fit best atm i'm more inclined for class reference than id's but that's just IMO.

**Review by marcmortensen — APPROVED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (mverdaguer) [reply .circleci/config.yml:40]: Sure, marcP is working on it on his branch :)

File: src/views/Books.vue (hunk 2 above)
Comment (mverdaguer) [reply src/views/Books.vue:2]: Done: https://github.com/orgs/lesfieres/projects/1#card-19548795

**Pull request merged**